official/transformer/v2/misc.py

```diff
@@ -195,8 +204,9 @@ def _check_export_vocab_file(flags_dict):
 def get_callbacks():
   """Returns common callbacks."""
   callbacks = []
-  time_callback = keras_utils.TimeHistory(FLAGS.batch_size, FLAGS.log_steps)
-  callbacks.append(time_callback)
+  if FLAGS.enable_time_history:
```

Comment:
can you verify no subsequent code fails if this is turned off (like building the stats etc)

Reply:
Actually, tranformer_main.py fails no matter whether it's turned off:

```
Exception ignored in: <bound method _CheckpointRestoreCoordinator.__del__ of <tensorflow.python.training.tracking.util._CheckpointRestoreCoordinator object at 0x7f8f8021d1d0>>
Traceback (most recent call last):
  File "/home/xunkai/anaconda3/envs/py36/lib/python3.6/site-packages/tensorflow/python/training/tracking/util.py", line 244, in __del__
    .format(pretty_printer.node_names[node_id]))
  File "/home/xunkai/anaconda3/envs/py36/lib/python3.6/site-packages/tensorflow/python/training/tracking/util.py", line 93, in node_names
    path_to_root[node_id] + (child.local_name,))
  File "/home/xunkai/anaconda3/envs/py36/lib/python3.6/site-packages/tensorflow/python/training/tracking/object_identity.py", line 76, in __getitem__
    return self._storage[self._wrap_key(key)]
KeyError: (<tensorflow.python.training.tracking.object_identity._ObjectIdentityWrapper object at 0x7f8f08621588>,)
```

Looks like irrelevant to this PR.

For benchmarking, maybe just don't turn off it; It's by default turned on.